README.md

```diff
@@ -13,18 +19,39 @@ npm install cf-buttons
 npm install cf-expandables
 ```
 
+### Project scaffolding
+
 Want some boilerplate?
+There is a yeoman generator through the
+[generator-cf](https://github.com/cfpb/generator-cf) project:
 
 ```sh
 npm install -g yo generator-cf
 mkdir my-new-project && cd $_
 yo cf
 ```
 
-| buttons | core | expandables | forms | grid | icons | layout | pagination | typography |
-|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
-| [![npm](https://img.shields.io/npm/v/cf-buttons.svg?style=flat-square)](https://www.npmjs.com/package/cf-buttons)  | [![npm](https://img.shields.io/npm/v/cf-core.svg?style=flat-square)](https://www.npmjs.com/package/cf-core)  | [![npm](https://img.shields.io/npm/v/cf-expandables.svg?style=flat-square)](https://www.npmjs.com/package/cf-expandables)  | [![npm](https://img.shields.io/npm/v/cf-forms.svg?style=flat-square)](https://www.npmjs.com/package/cf-forms)  | [![npm](https://img.shields.io/npm/v/cf-grid.svg?style=flat-square)](https://www.npmjs.com/package/cf-grid)  | [![npm](https://img.shields.io/npm/v/cf-icons.svg?style=flat-square)](https://www.npmjs.com/package/cf-icons)  | [![npm](https://img.shields.io/npm/v/cf-layout.svg?style=flat-square)](https://www.npmjs.com/package/cf-layout)  | [![npm](https://img.shields.io/npm/v/cf-pagination.svg?style=flat-square)](https://www.npmjs.com/package/cf-pagination)  | [![npm](https://img.shields.io/npm/v/cf-typography.svg?style=flat-square)](https://www.npmjs.com/package/cf-typography)  |
+### Using Less files directly
+
+If you don't want to use the
+[generator-cf](https://github.com/cfpb/generator-cf) yeoman generator,
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/351/commits/8293224f85be823317aa0844f8295a8803cc8b53